dramatiq/middleware/middleware.py

```diff
@@ -78,19 +86,19 @@ def after_declare_queue(self, broker, queue_name):
         them until messages are enqueued or consumed.
         """
 
-    def after_declare_delay_queue(self, broker, queue_name):
+    def after_declare_delay_queue(self, broker: Broker, queue_name: str) -> None:
         """Called after a delay queue has been declared."""
 
-    def before_enqueue(self, broker, message, delay):
+    def before_enqueue(self, broker: Broker, message: Message, delay: int) -> None:
         """Called before a message is enqueued."""
 
-    def after_enqueue(self, broker, message, delay):
+    def after_enqueue(self, broker: Broker, message: Message, delay: int) -> None:
         """Called after a message has been enqueued."""
 
-    def before_delay_message(self, broker, message):
+    def before_delay_message(self, broker: Broker, message: Message) -> None:
```

Comment:
I think `MessageProxy` for before_delay_message, before_process_message, after_process_message and after_skip_message